base/templates/base/public_base.html

```diff
@@ -429,20 +429,46 @@ <h3>{{self.friendly_name}}Social Media</h3>
                                 {% endif %}
                             </ul>
                         </div>
-                    {% endif %}
-
-                    {% if libra %}
+			{% endif %}
+			
+			{% if libra %}
```

Comment:
There are some indentation and whitespace problems in this template. This isn't the only place where it happens and I won't call them all out. It's not a big deal but we should fix it. Just block serves as a good example though. Line 432 has a closing if statement that doesn't line up with the opening if statement. 433 has empty whitespace on a newline and the html after line 434 is indented too much.

I don't believe we've always been consistent in all of our template files. Sometimes they use 4 space indentation and sometimes 2. I think 4 is more common on our site, and it's what's being used in this file, so let's go with that.